etc/internal_options.conf

```diff
@@ -359,6 +359,9 @@ wazuh_database.interval=60
 # 0. Use system default
 wazuh_database.max_queued_events=0
 
+# Vulnerability Detector task message queue size
+wazuh_vuln_detector.task_message_queue_size=10000
```

Comment:
Considering that the corresponding queue is called `vu_messages_queue`, I'd remove the word `task` from the setting name.

Reply:
In what issue will we attack the documentation of this internal configuration?

Reply:
@Dwordcito We have this tracking issue with all the required changes in the documentation related to the epic #9220 
https://github.com/wazuh/wazuh-documentation/issues/5186